cmd/vicadmin/usersession.go

```diff
@@ -86,9 +86,18 @@ func (u *UserSessionStore) VSphere(ctx context.Context, id string) (*session.Ses
 		return nil, fmt.Errorf("User session with unique ID %s does not exist", id)
 	}
 	if us.vsphere == nil {
-		return nil, fmt.Errorf("No vSphere session found. User with ID %s must authenticate again.", id)
+		return nil, fmt.Errorf("No vSphere session found for user with ID %s", id)
 	}
 
+	vsphus, err := us.vsphere.SessionManager.UserSession(ctx)
+	if err != nil || vsphus == nil {
+		u.Delete(id)
+		errMsg := fmt.Sprintf("Could not validate session for user %s; must log in again", id)
```

Comment:
I format the message w/o the error in the case where `err` == `nil` but `vsphus` is `nil`. If err is non-nil the err is appended. That is why I format first.